modules/network-observability-configuring-multicluster.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-multicluster_{context}"]
+= Configuring availability zones
+You can configure the `FlowCollector` to collect information about the cluster availability zone. This allows you to group by zones  
```

Comment:
```suggestion
You can configure the `FlowCollector` to collect information about the cluster availability zone. This allows you to enrich flows with Nodes "topology.kubernetes.io/zone" label value.
```

Can we also point the Kubernetes doc for this label here ?
https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone
Or maybe there is an Openshift equivalent ?

Reply:
I'm not sure if this is equivalent or not but I see [machine.openshift.io/zone-group](https://docs.openshift.com/container-platform/4.14/post_installation_configuration/aws-compute-edge-tasks.html#edge-machine-pools-aws-local-zones_aws-compute-edge-tasks) mentioned in the doc but not in any kind of reference like the Kubernetes link you provided. I think yeah I can link to this Kubernetes reference.